packages/manager/apps/dedicated/client/app/ip/components/list/popover/ip.html

```diff
@@ -291,7 +291,33 @@
             ></button>
         </li>
 
-        <!-- IP DELETE -->
+        <!-- PCC DELETE -->
+        <li data-ng-if="ipBlock.type == 'PCC'">
```

Comment:
I guess you can compare with the const that you create ( `export const PCC = 'PCC'` ) ?

Reply:
yes